packages/utilities/fast-web-utilities/src/rtl-scroll-converter.spec.ts

```diff
@@ -182,8 +182,8 @@ describe("RtlScrollConverter", (): void => {
     });
 
     it("applyDirectScrollConverters applies correct converters", () => {
-        RtlScrollConverter["getRtlScrollLeftConverter"] = null;
-        RtlScrollConverter["setRtlScrollLeftConverter"] = null;
+        RtlScrollConverter["getRtlScrollLeftConverter"] = RtlScrollConverter["reverseGetRtlScrollConverter"];
```

Comment:
Setting ```getRtlScrollLeftConverter``` and ```setRtlScrollLeftConverter``` to something other than the defaults (```directGetRtlScrollConverter``` and ```directSetRtlScrollConverter``` respectively)  to test the functionality of ```applyDirectScrollConverters()```